content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -293,6 +293,8 @@ insert into mydb.t1 values (3); -- modifying the `branch2` branch
 ### Options
 
 `-b`: Create a new branch with the given name.
+`-B`: Similar to `-b`, but will destructively move a branch if it already exists.
```

Comment:
(minor) You could probably drop "destructively"; it just sounds a little scary.

Reply:
it is scary, tho.